### ISSUE TITLE
Add support for H2O.ai models

### ISSUE THREAD

**Issue by sayakpaul** [bentoml/BentoML#92]
The H2O.ai implementations of linear models, tree-based models and monotonically constrained models are really good. Moreover, H2O.ai supports AutoML as well. 

Since BentoML has support for Scikit-Learn, Keras, TensorFlow and so son, I thought it might be a good idea to add H2O.ai models in this list.

Nevertheless, you guys are doing a pretty awesome job and I am preparing demo notebook to make myself accustomed with the workflow of BentoML. Will share once completed.

**Comment by parano** [comment]
Thanks @sayakpaul, it's a great idea to add H2O.ai model support, I'm adding it to our roadmap list. It seems straight forward to add a new artifact type from our side with h2o python module's save_model and load_model method.

Let us know how it goes with your demo notebook, would love to get your feedback.

**Comment by sayakpaul** [comment]
Yes, sure. It took me 7 minutes (yes I measured the time) to wrap Scikit-Learn model into a REST API using BentoML and then test it in Postman. I have pushed my notebook [here](https://github.com/sayakpaul/BentoML-Explorations) (I will fix the references in some time). I am now working on wrapping a tf-keras model for a sentiment classification problem (dataset is scraped from Zomato :P). Will share it once done.
